tests/common.go

```diff
@@ -1904,6 +1912,61 @@ func ValidateApplications(contexts []*scheduler.Context) {
 	})
 }
 
+// ValidateApplications validates applications
+func ValidateApplicationsStartData(contexts []*scheduler.Context, appContext context1.Context) (chan string, *errgroup.Group, map[string][]appDriver.ApplicationDriver) {
+
+	// Resetting the global map before starting the new App Validations
+	NamespaceAppWithDataMap = make(map[string][]appDriver.ApplicationDriver)
+
+	var allHandlers []appDriver.ApplicationDriver
+
+	Step("validate applications", func() {
```

Comment:
Do not use `Step` inside any helper functions. There is some issue when we move to ginkgo 2.0 when it is trying to build the spec tree.

Reply:
Done